bin/install.sh

```diff
@@ -11,7 +11,7 @@ hash npm &> /dev/null && npm install || {
 }
 
 echo "Looking for Alfred Preferences..."
-paths=("$1" "$HOME/Dropbox" "$HOME/Library/Mobile Documents/com~apple~CloudDocs" "$HOME/Library/Application Support/Alfred 3" "$HOME/Library/Application Support/Alfred 2")
+paths=("$1" "$HOME/Dropbox" "$HOME/Library/Mobile Documents/com~apple~CloudDocs" "$HOME/Library/Application Support/Alfred 3" "$HOME/Library/Application Support/Alfred 2" "$HOME/Google Drive")
```

Comment:
The paths in the `$paths` array are searched in the order of which they appear. It may make more sense to include the Google Drive directory prior to the default Alfred locations (in the `Library/Application Support` folder).